api_tests/nodes/views/test_node_contributors_list.py

```diff
@@ -577,6 +577,24 @@ def test_add_contributors_errors(
             expect_errors=True)
         assert res.status_code == 409
 
+    def test_unregistered_contributor_invalid_email(
```

Comment:
I'd add this test under the above `test_add_contributors_errors` instead of making it it's own test to save on setup time.   I don't go crazy with this - if it's a lot more clear to be a separate function or data is changing to much within a function, I make it it's own test, but if I can group it with another test, I go ahead and do that.

you would add a comment - `# test_unregistered_contributor_invalid_email` and then add your test beneath that.